Mathlib/RingTheory/PowerSeries/Order.lean

```diff
@@ -303,30 +311,58 @@ theorem order_X_pow (n : ℕ) : order ((X : R⟦X⟧) ^ n) = n := by
   rw [X_pow_eq, order_monomial_of_ne_zero]
   exact one_ne_zero
 
+@[simp]
+theorem divided_by_X_pow_order_of_X_eq_one :
```

Comment:
The name suggests the theorem says `X = 1 → divided_by_X_pow_order ...something...`. Probably you should just drop the `of_`. Oh, but you're only moving it. Followup, then.